src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ProjectCard.kt

```diff
@@ -0,0 +1,22 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import javafx.geometry.Pos
+import javafx.scene.control.Button
+import org.wycliffeassociates.otter.common.data.model.Project
+import org.wycliffeassociates.otter.common.ui.widgets.IProjectCard
+import tornadofx.*
+import tornadofx.Stylesheet.Companion.root
+
+class ProjectCard(project: Project) : Button() {
```

Comment:
a button is a control that you click, so the intent would be that the whole thing is a button. You can have image buttons, where part of the button has a picture, but the whole thing is just a button. That's not exactly what we want here though? Right now we have a couple labels, eventually it may be a label and an image, etc, but both then actually *contain* a button- implying the whole widget is not.

I'd extend VBox here, unless you know you want all of the APIs (or at least most of them) of the Button class.
(this also would mean you can pull the vbox out of the with root, since you become the vbox here)